src/main/java/org/folio/services/transactions/CreditService.java

```diff
@@ -29,10 +32,12 @@ public Future<Transaction> createTransaction(Transaction credit, RequestContext
 
   @Override
   public Future<Void> updateTransaction(Transaction credit, RequestContext requestContext) {
+    log.debug("updateTransaction:: Updating transaction '{}'", credit.getId());
     return Future.succeededFuture()
       .map(v -> {
         transactionService.validateTransactionType(credit, Transaction.TransactionType.CREDIT);
-        if (!Boolean.TRUE.equals(credit.getInvoiceCancelled())) {
+        if (Boolean.FALSE.equals(credit.getInvoiceCancelled())) {
+          log.warn("updateTransaction:: Credit invoice is not cancelled");
```

Comment:
for warn, error logs better to add some identifier, in our case creditId

Reply:
Thanks, added